src/fragment-components/a-icon.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from 'react';
+import { Search, Dropdown, ModalWrapper } from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { css } from 'emotion';
+import { ComponentInfo } from '.';
+import * as Icons from '@carbon/icons-react';
+import image from './../assets/component-icons/icons.svg';
+import { ElementTile } from '../components/element-tile';
+import isEmpty from 'lodash/isEmpty';
+import { angularClassNamesFromComponentObj, reactClassNamesFromComponentObj } from '../utils/fragment-tools';
+const searchStyle = css`
+	margin-top: 15px;
+`;
+
+const elementTileListStyle = css`
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	margin-top: 20px;
+`;
+
+const elementTileStyle = css`
+	border: 1px solid #d8d8d8;
+	min-width: 34px;
+	height: 30px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-bottom: 1rem;
+	flex-direction: column;
+`;
+
+const sizeItems = [
+	{ id: '16', text: 'Small' },
+	{ id: '20', text: 'Medium' },
+	{ id: '24', text: 'Large' },
+	{ id: '32', text: 'Extra large' }
+];
+
+const getIcons = () => {
+	const items: any = [];
+	Object.entries(Icons).forEach((item: any) => {
```

Comment:
```suggestion
	Object.entries(Icons).forEach(([ icon, size ]) => {
```
Probably not exact but something to that effect?

Reply:
I causes errors since not every object has a size object and we cannot we . operator for variables like `name`  because the type is unknown.